Bump HyperDualNumbers to v1.0.0

goedman opened the pull request:
Include Edwin’s fixes for asin, acos and atan.

tkelman commented:
Can you push the git tags so they show up at https://github.com/JuliaDiff/HyperDualNumbers.jl/releases ?

tkelman commented:
You should also enable Travis CI, it doesn't look like it's running.

goedman commented:
Thanks Tony!

On a long trip right now, but will make a patch release addressing your comments on my return to the US in August. Just wanted to get Edwin's fixes and a Julia 0.5-pre version out there.

Regards,
Rob

Sent from Rob Goedman's iPad Pro


> On Jul 13, 2016, at 06:23, Tony Kelman <notifications@github.com> wrote:
> 
> Merged #5597.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
